packages/k6-tests/config/fspiopDiscovery.json

```diff
@@ -4,11 +4,13 @@
       "executor": "ramping-vus",
       "exec": "fspiopDiscoveryScenarios",
       "env": {
-        "UNIDIRECTIONAL": "true"
+        "UNIDIRECTIONAL": "false"
       },
-      "startVUs": 1,
+      "startVUs": 5,
       "stages": [
-        { "duration": "10s", "target": 6 }
+        { "duration": "30s", "target": 600 },
+        { "duration": "1m", "target": 1000 },
+        { "duration": "2m", "target": 1000 }
```

Comment:
Can we leave the defaults as low? Because this testing stack is for everyone who wants to test performance with minimum machines and these high numbers may blow up their setup. We need to find a way to override these when we test with our large scale environment.

Reply:
I created a separate folder for k8s configs and env variable `K6_SCRIPT_CONFIG_FOLDER_NAME` to use that folder. If the variable is not set, the default `config` folder is used.